src/test/java/uk/gov/companieshouse/mapper/AppealMapperTest.java

```diff
@@ -103,7 +104,7 @@ void shouldMapValueWhenValueIsNotNull() {
                             TestData.Appeal.Reason.Attachment.contentType,
                             TestData.Appeal.Reason.Attachment.size
                         ))
-                    )
+                    ), null
```

Comment:
Where's the unit tests for mapping an Appeal with an illness reason?

Reply:
Resolved on https://github.com/companieshouse/lfp-appeals-api/pull/112